errorDisplay.js

```diff
@@ -8,60 +8,99 @@ define(function (require, exports, module) {
         ExtensionUtils = brackets.getModule("utils/ExtensionUtils"),
         lineWidgetHTML = require("text!inlineWidget.html"),
         currentErrorWidget;
-
+    var errorToggle;
+    var isShowingDescription;
+    var errorObject;
+    
     ExtensionUtils.loadStyleSheet(module, "main.less");
     require("tooltipsy.source");
 
+    //Publicly available function to remove all errors from brackets
+    function cleanUp(line) {
+        removeButton();
+        removeHighlight(line);
+        hideDescription();
+        isShowingDescription = false;
+    }
+
+    //Publicly available function used to display error markings
+    function scafoldHinter(errorStart, errorEnd, errorObj) {
+        //Setup neccessary variables
```

Comment:
Comments like these aren't useful.  You can remove this, and the ones on lines 33 and 36, since the naming of the functions accomplishes the same goal.